manifests/profile/web.pp

```diff
@@ -95,7 +95,11 @@
   ## optionally manage the SSL certificate used by nginx
   if $ssl_cert_manage {
     ## Generate SSL certificates
-    $_ssl_subj = "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information Technology/CN=${$trusted['certname']}"
+    $_truncated_certname = $trusted['certname'].length > 64 ? {
```

Comment:
please put a comment in here about why we're doing this

Reply:
thanks, added a comment for it